packages/features/portfolio/src/components/total-earnings-tooltip/total-earnings-tooltip.tsx

```diff
@@ -10,10 +10,12 @@ export interface TotalEarningsTooltipProps {
   toolTipData: {
     nonNoteEarnings?: string;
     noteEarnings?: string;
-    underlyingBaseCurrency?: string;
-    underlying?: string;
-    noteBaseCurrency?: string;
-    note?: string;
```

Comment:
If there's no special treatment of each these strings, can we make this more readable by just taking two props:

Something like:
```
perAssetEarnings: { underlying: string, baseCurrency: string }[]
totalEarnings: { totalBaseEarnings: string, totalIncentiveEarningsInBase?: string[] }
```